src/content/docs/en/pages/main-menu/reference/store/object-storage/index.mdx

```diff
@@ -0,0 +1,126 @@
+
+For instance, the list of keys below represents the simulated hierarchy of an application stored in a bucket with prefixes:
+
+```bash
+README.md
+/src/index.js
+/src/index.html
+/src/assets/styles.css
+/src/assets/images/image.png
+```
+
+The object `README.md` is located at the root of the bucket. The prefix `src` corresponds to a folder and contains the objects `index.js` and `index.html`. Additionally, the `src` prefix also contains the `assets` prefix that houses a `styles.css` object and the `images` prefix, which, in turn, houses the `image.png` object.
+
+Uploading a different object or modifying object contents using an existing key replaces the object. Once an object is replaced, earlier versions can't be retrieved.
+
+---
+
+## Origin
+
+With **Object Storage**, you can use buckets as an [origin](/en/documentation/products/build/edge-application/origins/#object-storage) in Azion **Edge Application** to retrieve the content of an edge application.
+
+You can determine if the content is retrieved from the root of the bucket or from a prefix within the bucket.
+
+---
+
+## Transactions
+
+A transaction refers to an exchange between a client and **Object Storage**. Any action related to buckets and objects (create, delete, list, and update) is considered a transaction. Each time one of these methods is used, a transaction is logged. 
+
+:::note
+Transactions are charged by classes. If a transaction falls outside of the scope of class A operations, it'll be billed to your account. For more information on transaction and class charges, see the [Pricing](https://www.azion.com/en/pricing/) page.
+:::
+
+Transaction history can be accessed through [Real-Time Events GraphQL API](/en/documentation/products/graphql-api/features/events-fields/#activityhistoryevents-activity-history).
+
+The current release of the **Object Storage** offers the following transactions:
+
+| Class | Transaction name | Description |
+| --- | --- | --- |
+| A | Put Object | Upload an object to a bucket. Executed through the `PUT` method. <br></br>Requires read-write permission. |
+| A | Post Object | Upload an object to a bucket. Executed through the `POST` method. <br></br>Requires read-write permission. |
+| A | Delete Object | Remove an object from a bucket. Executed through the `DELETE` method. <br></br>Requires read-write permission. |
+| A | Delete Bucket | Remove a bucket from an account. Executed through the `DELETE` method. |
+| B | Get Object | Retrieve an object from a bucket. Executed through the `GET` method. |
+| C | List Objects | Retrieve a list of objects uploaded to a bucket. Executed through the `GET` method. |
+| C | Create Bucket | Create a new bucket for an account. Executed through the `POST` method. |
+| C | List Buckets | Retrieve a list of buckets associated with an account. Executed through the `GET` method. |
+| C | Update Bucket | Modify bucket information. Executed through the `PATCH` method. |
```

Comment:
```suggestion
| A | Put Object | Upload an object to a bucket. Executed through the `PUT` method. <br></br>Requires read-write permission |
| A | Post Object | Upload an object to a bucket. Executed through the `POST` method. <br></br>Requires read-write permission |
| A | Delete Object | Remove an object from a bucket. Executed through the `DELETE` method. <br></br>Requires read-write permission |
| A | Delete Bucket | Remove a bucket from an account. Executed through the `DELETE` method |
| B | Get Object | Retrieve an object from a bucket. Executed through the `GET` method |
| C | List Objects | Retrieve a list of objects uploaded to a bucket. Executed through the `GET` method |
| C | Create Bucket | Create a new bucket for an account. Executed through the `POST` method |
| C | List Buckets | Retrieve a list of buckets associated with an account. Executed through the `GET` method |
| C | Update Bucket | Modify bucket information. Executed through the `PATCH` method |
```

Reply:
Description should be in 3rd person?
Uploads, Removes, Retrieves, etc... Considering it refers to transactions and the actions they do....